src/Agent/NewRelic/Agent/Parsing/SqlParser.cs

```diff
@@ -129,6 +131,16 @@ public static ParsedSqlStatement GetParsedDatabaseStatement(DatastoreVendor data
                 // Remove comments.
                 var statement = CommentPattern.Replace(commandText, string.Empty).TrimStart();
 
+                if (!IsSingleSqlStatement(statement))
+                {
+                    // Remove leading SET commands
+
+                    // Trimming any trailing semicolons is necessary to avoid having the LeadingSetPattern
+                    // match a SQL statement that ONLY contains SET commands, which would leave us with nothing
+                    statement = statement.TrimEnd(';');
```

Comment:
If you could make ';' a constant, it would be great.

Reply:
Done.